### PR TITLE
ftx: add basic exchange order execution limits for lower bound

### PR DESCRIPTION
**Pull request by shazbert**
# PR Description

Adds in step increments for price and amount and amount minimum to reduce request errors from API endpoint. 

Fixes # (issue)

## Type of change

Please delete options that are not relevant and add an `x` in `[]` as item is complete.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## How has this been tested

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration and
also consider improving test coverage whilst working on a certain feature or package.

- [ ] go test ./... -race
- [ ] golangci-lint run
- [ ] Test X

## Checklist

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation and regenerated documentation via the documentation tool
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally and on Github Actions/AppVeyor with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: exchanges/ftx/ftx.go [hunk 1]
```diff
@@ -1378,3 +1379,46 @@ func (f *FTX) SubaccountTransfer(coin currency.Code, source, destination string,
...
+			return nil, fmt.Errorf("unhandled data type %s, cannot process exchange limit",
+				data[x].MarketType)
+		}
+
+		limits = append(limits, order.MinMaxLevel{
+			Pair:       cp,
+			Asset:      a,
+			StepPrice:  data[x].PriceIncrement,
+			StepAmount: data[x].SizeIncrement,
+			MinAmount:  data[x].SizeIncrement,
```
Comment (thrasher-) [inline exchanges/ftx/ftx.go:1420]: Thoughts on using MinProvideSize here? Seems to be the same as SizeIncrement judging by API output but seems more correct

**Review by thrasher- — CHANGES_REQUESTED** [review]

File: exchanges/ftx/ftx.go (hunk 1 above)
Comment (shazbert) [reply exchanges/ftx/ftx.go:1420]: 🦅 Will fix! Yeah there are some differences between currencies.

**Review by thrasher- — APPROVED** [review]
Thanks for making those changes!

File: exchanges/ftx/ftx.go [hunk 2]
```diff
@@ -1378,3 +1379,46 @@ func (f *FTX) SubaccountTransfer(coin currency.Code, source, destination string,
 	}
 	return &resp.Data, nil
 }
+
+// FetchSpotExchangeLimits fetches spot order execution limits
```
Comment (gloriousCode) [inline exchanges/ftx/ftx.go:1383]: `FetchExchangeLimits`

**Review by gloriousCode — CHANGES_REQUESTED** [review]
Just one ABSOLUTELY CRITICAL ERROR MADE!

**Review by gloriousCode — APPROVED** [review]
tACK! 🎉 🎉 🎉 🎉

**Pull request merged**